src/main/webapp/app/lecture/lecture.component.ts

```diff
@@ -111,6 +120,17 @@ export class LectureComponent implements OnInit {
         this.applyFilters();
     }
 
+    sortRows() {
+        //Sort alphabetically for title and description
+        if (this.predicate == 'title')
+            this.filteredLectures.sort((a: Lecture, b: Lecture) => (this.ascending ? (a.title || '').localeCompare(b.title || '') : (b.title || '').localeCompare(a.title || '')));
+        else if (this.predicate == 'description')
+            this.filteredLectures.sort((a: Lecture, b: Lecture) =>
+                this.ascending ? (a.description || '').localeCompare(b.description || '') : (b.description || '').localeCompare(a.description || ''),
+            );
```

Comment:
How do we sort by course? Do we sort by id? That would be unintuitive as the Coursename is displayed.

Reply:
Thank you for pointing that out. You're correct, sorting by the course ID would indeed be unintuitive as the course name is what's displayed to the user. So the Course sorting option is unnecessary and will be removed.